IPython/core/tests/test_interactiveshell.py

```diff
@@ -143,26 +143,13 @@ def __repr__(self):
 
     def test_future_flags(self):
         """Check that future flags are used for parsing code (gh-777)"""
```

Comment:
This test no longer actually tests the thing it was meant to test.

There is a hidden future import that we can use for testing, though it may be CPython-only:

```python
from __future__ import barry_as_FLUFL
1 <>  2   # This is now valid syntax
```